src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractReportServiceImpl.java

```diff
@@ -477,6 +480,19 @@ public void setReportValidationErrorsSubTitle(String reportValidationErrorsSubTi
         this.reportValidationErrorsSubTitle = reportValidationErrorsSubTitle;
     }
 
+    public String getReportValidationErrorsSubTitleByPassedTransactionNote() {
+        if (StringUtils.isEmpty(reportValidationErrorsSubTitle)) {
```

Comment:
Should this condition be checking the "reportValidationErrorsSubTitleByPassedTransactionNote" field instead?

Reply:
thanks, yes